sharding-scaling/sharding-scaling-bootstrap/src/main/java/org/apache/shardingsphere/shardingscaling/Bootstrap.java

```diff
@@ -49,72 +41,36 @@
 @Slf4j
 public class Bootstrap {
 
+    private static final int PORT = Integer.parseInt(System.getProperty("port", "8080"));
+
```

Comment:
Moving port to configFile maybe better. You can refer `Bootstrap in sharding-proxy`.

Reply:
I will submit a new issue for config related, so here will stay as it is?